### PR TITLE
Catch passing duplicate block to arbitrate consensus fault

### PR DESCRIPTION
**Pull request by whyrusleeping**
Reviewers: Please verify that nothing else is missing. This check wasnt in the spec, there may be others.

### CODE REVIEW

File: build/params_shared.go [hunk 1]
```diff
@@ -105,3 +105,9 @@ const BlsSignatureCacheSize = 40000
 // Limits
 
 const BlockMessageLimit = 512
+
+/////////////
+// Fork Heights
+//
+
+const ForkBlizzardHeight = 6000
```
Comment (magik6k) [inline build/params_shared.go]: `build/forks.go`

File: chain/actors/actor_storagepower.go [hunk 2]
```diff
@@ -138,6 +138,12 @@ func (spa StoragePowerActor) ArbitrateConsensusFault(act *types.Actor, vmctx typ
 		return nil, aerrors.New(2, "blocks must be from the same miner")
 	}
 
+	if vmctx.BlockHeight() > build.ForkBlizzardHeight {
```
Comment (magik6k) [inline chain/actors/actor_storagepower.go]: Comment as fork, or use withUpdates, creating v0/v1 methods

Comment (magik6k) [reply chain/actors/actor_storagepower.go]: (from `chain/actors/forks.go`)

**Comment by whyrusleeping** [comment]
Merged as part of #1100

**Pull request closed without merging**